obs-studio-server/source/nodeobs_configManager.cpp

```diff
@@ -172,6 +172,11 @@ void initBasicDefault(config_t* config)
 	config_set_default_string(config, "Video", "ColorSpace", "601");
 	config_set_default_string(config, "Video", "ColorRange", "Partial");
 
+	// Set default value for EnableGPUAsRenderDevice
+	if (!config_has_user_value(config, "Video", "ForceGPUAsRenderDevice")) {
+		config_set_bool(config, "Video", "ForceGPUAsRenderDevice", true);
```

Comment:
Sorry I miss this during my first review but I would use `config_set_default_bool` here in any cases, this is what it is meant for.

Reply:
I did the change and tested everything. It is working successfully.